### PR TITLE
Feature/improve autohelp

### PR DESCRIPTION
**Pull request by DRSchlaubi**

### CODE REVIEW

File: src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/ContentFetcher.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            .url(url)
+            .get()
+            .build()
+        return httpClient.newCall(request).executeAsync().thenApply { response ->
+            response.body.use {
+                it?.string()
+            }
+        }
+    }
+}
```
Comment (daspositioned) [inline src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/ContentFetcher.kt]: Missing newline

File: src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/LanguageGusser.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    override fun onPushStyle(style: String?) = Unit
+
+    override fun onPushCodeBlock(codeLexeme: CharSequence?) = Unit
+
+    override fun onAbort(code: CharSequence?) = Unit
+
+    override fun onStart() = Unit
+
+}
```
Comment (daspositioned) [inline src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/LanguageGusser.kt]: Missing newline

File: src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/AutoHelp.kt [hunk 3]
```diff
@@ -30,84 +28,87 @@ import com.github.seliba.devcordbot.dsl.sendMessage
...
     private val blacklist: List<String>,
     knownLanguages: List<String>
 ) {
-    private val languageGuesser = Highlighter(UselessRendererFactoryThing())
-    private val knownLanguages = knownLanguages.toTypedArray()
 
-    /**
-     * Listens for new messages.
-     */
+    private val gussr = LanguageGusser(knownLanguages)
```
Comment (daspositioned) [inline src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/AutoHelp.kt]: You may fix the typo

Comment (DRSchlaubi) [reply src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/AutoHelp.kt]: https://www.geoguessr.com/ triggered me too much with this name

Comment (jkoenig134) [reply src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/AutoHelp.kt]: This is obviously not resolved.

File: src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/AutoHelp.kt [hunk 4]
```diff
@@ -206,9 +198,9 @@ class CommonPitfallListener(
 
         // https://regex101.com/r/N8NBDz/1
         private val PASTEBIN_PATTERN = "(?:https?:\\/\\/(?:www\\.)?)?pastebin\\.com\\/(?:raw\\/)?(.*)".toRegex()
+
+        private const val BYPASS_WORD = "_Ü?"
```
Comment (jkoenig134) [inline src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/AutoHelp.kt]: Get BYPASS_WORD from config

**Review by jkoenig134 — APPROVED** [review]

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**